android/build.gradle

```
@@ -44,15 +44,14 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
-    implementation 'io.sentry:sentry-android:6.4.3'
+    implementation 'io.sentry:sentry-android:6.9.2-tobeupdated'//TODO
```

Update when the Android SDK gets updated
```suggestion
    implementation 'io.sentry:sentry-android:6.9.2'
```